ambry-network/src/test/java/com.github.ambry.network/BlockingChannelConnectionPoolTest.java

```diff
@@ -377,7 +387,7 @@ public void testBlockingChannelConnectionPool()
     props.put("connectionpool.max.connections.per.port.ssl", "5");
     ConnectionPool connectionPool =
         new BlockingChannelConnectionPool(new ConnectionPoolConfig(new VerifiableProperties(props)), sslConfig,
-            new MetricRegistry());
+            plainTextClusterMapConfig, new MetricRegistry());
```

Comment:
the other test is in disabled state due to errors - just made the change anyway to ensure that we do the right thing when we get around to fixing that.